stripe-test-e2e/src/test/java/com/stripe/android/test/e2e/Settings.kt

```diff
@@ -14,7 +14,9 @@ internal data class Settings(
         getMetadata(context, METADATA_KEY_BACKEND_URL_KEY)
     )
 
-    private companion object {
+    companion object {
+        const val PUBLISHABLE_KEY = "pk_test_ErsyMEOTudSjQR8hh0VrQr5X008sBXGOu6"
```

Comment:
Can we pass this via env variable instead?

Reply:
We just need to modify the `STRIPE_EXAMPLE_PUBLISHABLE_KEY` secret in https://github.com/stripe/stripe-android/settings/secrets/actions for this.

Reply:
Better yet would be to receive the publishable key from the server in the response to the first API call. less variables to keep track of in the client. We already do that on the PaymentSheet example app.

Reply:
This will require some changes on the backend app. I will keep it the way it is for now, but this is a nice improvment.